### ISSUE TITLE
No longer outputs a full library

### ISSUE THREAD

**Issue by k2snowman69** [rollup/plugins#215]
- Rollup Plugin Name: @rollup/plugin-node-resolve
- Rollup Plugin Version: 7.1.0 & 7.1.1
- Rollup Version: 1.27.14
- Operating System (or Browser): OSX
- Node Version: 10.16.3

### How Do We Reproduce?
1. git clone https://github.com/k2snowman69/bug-rollup-node-resolve
2. cd bug-rollup-node-resolve
3. npm install
4. npm run build

### Expected Behavior
Can be seen after npm installing `@rollup/plugin-node-resolve@7.0.0` where `dist/index.es.js` contains the full library output:
```
class ExampleComponent extends React.Component {
  sayHi() {
    console.log('hi');
  }
}

export { ExampleComponent };
```

### Actual Behavior
Can be seen after npm installing `@rollup/plugin-node-resolve@7.1.0` where `dist/index.es.js` contains only the output for the input file:
```
export { ExampleComponent } from "./component";
```

**Comment by shellscape** [comment]
@k2snowman69 thanks for opening an issue. Please try to distill your reproduction down further to a more minimal state. We encourage people to use the REPL option as that's minimal by default. Distilling your reproduction will help identify if the problem is inherent in a core plugin, rollup core, or a third party plugin / configuration.

**Comment by k2snowman69** [comment]
@shellscape Thanks for getting back to me so quickly and thanks for the suggestion (it was late, wasn't thining fully).

After following your suggestion, I narrowed the problem down to the `resolveOnly` flag in node-resolve. I believe previously this flag only applied to external packages but since 7.1.0 seems to be including internal packages as well.

In the REPL how do I change the config being passed into node-resolve? For now I've updated my github repo to be smaller and added a comment above the `resolveOnly` property
https://github.com/k2snowman69/bug-rollup-node-resolve/blob/master/rollup.config.js#L18
